docs/CHANGELOG.asciidoc

```diff
@@ -570,6 +571,43 @@ Search::
 * Fixes an issue in Search Playground where context limit errors were not handled well when using the Elastic Managed LLM ({kibana-pull}225360[#225360]).
 * Adjusts the `z-index` of the app menu header to not conflict with the Persistent Console ({kibana-pull}224708[#224708]).
 
+[[release-notes-8.18.6]]
+== {kib} 8.18.6
+
+The 8.18.6 release includes the following fixes.
+
+[float]
+[[enhancement-v8.18.6]]
+=== Enhancements
+Kibana security::
+* Improves error messages when your {kib} session fails to refresh a token ({kibana-pull}231118[#231118]).
+Machine Learning::
+* Improves error handling in Anomaly Explorer when fetching table data fails and category definition fails ({kibana-pull}231779[#231779]).
+
+[float]
+[[fixes-v8.18.6]]
+=== Fixes
+Alerting::
+* Fixes an issue where maintenance windows with scoped queries were not being correctly applied to all rule types ({kibana-pull}232307[#232307]).
+Cases::
```

Comment:
Since the Security team now owns Cases, I think we need to move these PRs over to those release notes. I pinged the PR owner about applying the team Security label to both his PRs.

```suggestion
```